src/Sylius/Bundle/UiBundle/Resources/translations/messages.pl.yml

```diff
@@ -491,6 +491,7 @@ sylius:
         meta_keywords: 'Słowa kluczowe w sekcji meta'
         metadata_index: 'Informacje o dokumencie'
         method: 'metoda'
+        missing_translation: 'brakujące tłumaczenie'
```

Comment:
We should not provide any non-English translation - Crowdin would take care of that 🖖